serviceexample.gitlab-ci.yml

```diff
@@ -0,0 +1 @@
+include: https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/belt-node8-service.gitlab-ci.yml
```

Comment:
Not sure this is how we'd do it at this point, is it? I suppose we still could, but this doesn't pull together all the new stuff you just wrote and it wouldn't really need to be documented in your new stuff.